visualstudio/2017/netstandard/NetstandardModuleSample/NetstandardModuleSample/module_dev_sample.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+    "loaders": [
+        {
+            "type": "dotnetcore",
+            "name": "dotnetcore",
+            "configuration": {
+                "binding.path": "dotnetcore.dll",
+                "binding.coreclrpath": "C:\\Program Files\\dotnet\\shared\\Microsoft.NETCore.App\\1.1.1\\coreclr.dll",
+                "binding.trustedplatformassemblieslocation": "C:\\Program Files\\dotnet\\shared\\Microsoft.NETCore.App\\1.1.1\\"
+            }
+        }
+    ],
+    "modules": [
+        {
+            "name": "logger",
+            "loader": {
+                "name": "native",
+                "entrypoint": {
+                    "module.path": "logger.dll"
+                }
+            },
+            "args": {
+                "filename": "log.txt"
+            }
+        },
+        {
+            "name": "dotnet_sensor_module",
+            "loader": {
+                "name": "dotnetcore",
+                "entrypoint": {
+                    "assembly.name": "NetstandardModuleSample",
+                    "entry.type": "SensorModule.DotNetSensorModule"
+                }
+            },
+            "args": "module configuration"
+        },
+        {
+            "name": "dotnet_sample_module",
+            "loader": {
+                "name": "dotnetcore",
+                "entrypoint": {
+                    "assembly.name": "NetstandardModuleSample",
+                    "entry.type": "NetstandardModuleSample.SampleModule"
+                }
+            },
+            "args": "module configuration"
+        }
+    ],
+    "links": [
+        {
+            "source": "dotnet_sensor_module",
+            "sink": "dotnet_sample_module"
+        }
+    ]
+}
```

Comment:
Again, this was created by the SDK team I imported it into the project from the iot-edge repo directly.